lib/wallet/api/http/Cardano/Wallet/Launch/Cluster.hs

```diff
@@ -907,41 +925,34 @@ generateGenesis dir systemStart initialFunds addPoolsToGenesis = do
     let startTime = round @_ @Int . utcTimeToPOSIXSeconds $ systemStart
     let systemStart' = posixSecondsToUTCTime . fromRational . toRational $ startTime
 
-    let pparams = Ledger.ShelleyPParams
-            { _minfeeA = 100
-            , _minfeeB = 100_000
-            , _minUTxOValue = Ledger.Coin 1_000_000
-
-            , _keyDeposit = Ledger.Coin 1_000_000
-            , _poolDeposit = Ledger.Coin 0
-
-            , _maxBBSize = 239_857
-            , _maxBHSize = 217_569
-            , _maxTxSize = 16_384
-
-            , _minPoolCost = Ledger.Coin 0
-
-            , _extraEntropy = Ledger.NeutralNonce
-
+    let pparams =
+            Ledger.emptyPParams
```

Comment:
We now won't be notified if a new field is added which we need to provide a more sensible value for what `emptyPParams` does. Looking at `emptyPParams` it does seem some values are more sensible than just "0" though, so hopefully this isn't a problem we're going to run into in the future.

https://github.com/input-output-hk/cardano-ledger/blob/2a03a3ee0b8b401552e985b7035c3dc382ff2f29/eras/babbage/impl/src/Cardano/Ledger/Babbage/PParams.hs#L317-L335